### PR TITLE
update ux

### PR DESCRIPTION
**Pull request by bieg**
ux update

### CODE REVIEW

**Comment by bieg** [comment]
revert bg

File: src/components/NumberedInfo.vue [hunk 1]
```diff
@@ -39,13 +39,14 @@ export default {
     h2 {
       @include font-size(2.6);
 
-      margin: 6px 0 0;
+      margin: 0.75em 0 0.15em 0;
```
Comment (mmeester) [inline src/components/NumberedInfo.vue]: shouldn't we use rem? I suggest introducing a `rem-calc` function

File: src/components/TwoLadies.vue [hunk 2]
```diff
@@ -31,22 +31,31 @@ export default {
...
-  p {
+  h2 {
     position: absolute;
     right: 0;
     bottom: 5%;
     left: 0;
     padding-bottom: 16px;
+    font-size: 3em;
+    font-weight: 700;
+    line-height: 1.1;
```
Comment (mmeester) [inline src/components/TwoLadies.vue]: let's stick to 1 then

File: src/views/Entrepreneur.vue [hunk 3]
```diff
@@ -72,14 +73,15 @@
       />
 
       <div v-show="addressLoaded" id="address">
-        <p>
+        <p class="adres_update">
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: `adress_update` -> aren't we using Bem?

File: src/design/_fonts.scss [hunk 4]
```diff
@@ -1,6 +1,8 @@
-$system-default-font-family: -apple-system, 'BlinkMacSystemFont', 'Segoe UI',
-  'Helvetica', 'Arial', sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
-  'Segoe UI Symbol';
+// $system-default-font-family: -apple-system, 'BlinkMacSystemFont', 'Segoe UI',
+//   'Helvetica', 'Arial', sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
+//   'Segoe UI Symbol';
+
+$system-default-font-family: 'Work Sans';
```
Comment (mmeester) [inline src/design/_fonts.scss]: Aren't we using fallback fonts here? Or is this done somewhere else?

File: src/views/Entrepreneur.vue [hunk 5]
```diff
@@ -12,14 +13,14 @@
       id="tegoedje"
       alt
       role="presentation"
-      src="../assets/img/bijnagoed.svg"
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: Aren't we giving feedback anymore with the logo?

Comment (bieg) [reply src/views/Entrepreneur.vue]: not like this i wouldnt - doesnt look profi

File: src/design/_fonts.scss (hunk 4 above)
Comment (bieg) [reply src/design/_fonts.scss]: ok will add fallback of sans-serif just to be sure

File: src/views/Entrepreneur.vue (hunk 5 above)
Comment (mmeester) [reply src/views/Entrepreneur.vue]: Who's the decision taker on this @jeroensmeets?

File: src/views/Entrepreneur.vue (hunk 3 above)
Comment (bieg) [reply src/views/Entrepreneur.vue]: ok

File: src/components/TwoLadies.vue (hunk 2 above)
Comment (bieg) [reply src/components/TwoLadies.vue]: there is a difference between 1 and 1.1

File: src/components/NumberedInfo.vue (hunk 1 above)
Comment (bieg) [reply src/components/NumberedInfo.vue]: not my call but sounds like a plan

**Comment by bieg** [comment]
updated

File: src/views/Entrepreneur.vue [hunk 6]
```diff
@@ -97,8 +99,12 @@
         field-label="IBAN nummer"
       />
 
-      <div v-if="step === 6">
-        <p>checkbox here to accept terms</p>
+      <div v-if="step === 6" class="avw">
+        <!-- <input id="checkbox" v-model="checked" type="checkbox" />
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: Why is this commented out?

File: src/views/Entrepreneur.vue [hunk 7]
```diff
@@ -149,21 +156,26 @@ export default {
         email: '',
         iban: '',
         address: {},
+        avw: '',
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: Name this conditions, i really had to think hard that this stands voor algemene voorwaarden

File: src/views/Entrepreneur.vue [hunk 8]
```diff
@@ -97,8 +99,12 @@
         field-label="IBAN nummer"
       />
 
-      <div v-if="step === 6">
-        <p>checkbox here to accept terms</p>
+      <div v-if="step === 6" class="avw">
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: Provide proper English classnames please

File: src/views/Entrepreneur.vue [hunk 9]
```diff
@@ -149,21 +156,26 @@ export default {
...
         address: {},
+        avw: '',
       },
     }
   },
   computed: {
+    checkedComputed() {
+      return this.checkedNames
+    },
+
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: What is the purpose of checkednames?

File: src/views/Entrepreneur.vue [hunk 10]
```diff
@@ -261,8 +273,18 @@ export default {
...
           return true
+
+        // case 7:
+        //   if (!AVW.isTrue(this.customer.avw)) {
+        //     this.showError(
+        //       'Klik nog wel even de Algemene Voorwaarden aan om verder te kunnen.'
+        //     )
+        //     return false
+        //   }
+        //   return true
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: Why commented?

File: src/views/Entrepreneur.vue [hunk 11]
```diff
@@ -335,18 +358,38 @@ export default {
...
+    left: 46px;
+
+    &:hover {
+      cursor: pointer;
+
+      &::after {
+        position: absolute;
+        bottom: 1.35em;
+        left: 1em;
+        z-index: 98;
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: 98? Can’t this be lower?

File: src/views/Entrepreneur.vue [hunk 12]
```diff
@@ -335,18 +358,38 @@ export default {
...
+        display: block;
+        padding: 0.3em 1em;
+        font-size: 12px;
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: No px for font sizes

File: src/views/Entrepreneur.vue [hunk 13]
```diff
@@ -335,18 +358,38 @@ export default {
...
+        border-radius: 0.5em;
+      }
+    }
   }
 
   #tegoedje {
-    max-height: 65px;
-    margin: 2em 0 3em;
+    width: 30%;
+    // max-height: 65px;
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: Remove

File: src/views/Entrepreneur.vue [hunk 14]
```diff
@@ -358,8 +401,22 @@ export default {
...
+    margin: 0 auto;
+    font-size: 1.9rem;
+    line-height: 1.05;
+  }
+
+  .avw {
+    width: 75%;
+    margin: 0 auto 2em auto;
+    font-size: 1.8rem;
+    line-height: 1.05;
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: Stick with 1

File: src/views/Entrepreneur.vue [hunk 15]
```diff
@@ -358,8 +401,22 @@ export default {
     }
   }
 
+  p.adres_update {
+    width: 85%;
+    margin: 0 auto;
+    font-size: 1.9rem;
+    line-height: 1.05;
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: Stick with 1

File: src/views/Home.vue [hunk 16]
```diff
@@ -4,26 +4,36 @@
...
+      class="start_btn"
+      linktext="Ja, dit wil ik ook"
+    />
+
+    <!-- <ButtonLink
+      link="iwillhelp"
+      class="disabled "
+      disabled="validated == 1"
+      linktext="Ja, ik wil helpen"
+    /> -->
```
Comment (mmeester) [inline src/views/Home.vue]: Remove the button

File: src/views/Entrepreneur.vue (hunk 8 above)
Comment (mmeester) [reply src/views/Entrepreneur.vue]: it's `address` and if we're using BEM don't do make it `address__update`

File: src/views/Entrepreneur.vue [hunk 17]
```diff
@@ -156,7 +158,7 @@ export default {
         email: '',
         iban: '',
         address: {},
-        avw: '',
+        checked: false,
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: what is checked? Sorry for being picky on this but if we want to make this easy to transfer between devs we need to have good declarative variables

File: src/views/Entrepreneur.vue [hunk 18]
```diff
@@ -72,14 +73,15 @@
       />
 
       <div v-show="addressLoaded" id="address">
-        <p>
+        <p class="adres--update">
```
Comment (mmeester) [inline src/views/Entrepreneur.vue]: it's `address` and if we're using BEM  why don't do make it `address__update`?

**Comment by IAfanasovMob** [comment]
I will create a new PR after rebase

**Pull request closed without merging**